### PR TITLE
Ag8 196 single vanities button test

### PR DESCRIPTION
**Pull request by nursena-tekoglu**

### CODE REVIEW

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+//     Background: User is on the home page
+//         Given User navigates to Home Page
+
+//     Scenario: Clicking the "Single Vanities" button redirects to Single Bathroom Vanities page
+//         When User clicks to Single Vanities
+//         Then User verifies the page title is "Single Bathroom Vanities"
+
+
+Given('User navigates to Home Page', () => {
+  cy.visit(Cypress.env('avaluxury_endpoint'));
```
Comment (KubraSA) [inline cypress/integration/step_definitions/Ui/single_vanities.cy.js:18]: Why didn't you use the one in background?

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+let HomePage=null
+before(()=>{
+    HomePage =new homePage();
+})
+
+When('User clicks to Single Vanities', () => {
+    HomePage.clickSingleVanities();
+});
+   
+Then('User verifies the page title is "Single Bathroom Vanities"', (pageTitle) => {
```
Comment (KubraSA) [inline cypress/integration/step_definitions/Ui/single_vanities.cy.js:30]: There is a function for verifying page title in commonsteps, pls use that one

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js (hunk 1 above)
Comment (nursena-tekoglu) [reply cypress/integration/step_definitions/Ui/single_vanities.cy.js:18]: I will remove this code from here then.

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js (hunk 2 above)
Comment (nursena-tekoglu) [reply cypress/integration/step_definitions/Ui/single_vanities.cy.js:30]: Actually I used that one just modified, will work on that one again

**Comment by nursena-tekoglu** [comment]
Codes are revised, please review

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+
+import { Given, When } from "cypress-cucumber-preprocessor/steps";
+import { homePage } from "../../../pages/homePage";
+import { SingeVanitiesPage } from "../../../pages/single_vanities_page";
+
+
+//     Background: User is on the home page
+//         Given User navigates to Home Page
+
```
Comment (KubraSA) [inline cypress/integration/step_definitions/Ui/single_vanities.cy.js:11]: why do we have these commented lines here? :)

Comment (nursena-tekoglu) [reply cypress/integration/step_definitions/Ui/single_vanities.cy.js]: They were helping me :) okay, removed them, will try to be more professional :)

**Review by KubraSA — APPROVED** [review]

**Pull request merged**